daemon/api_console_conf.go

```diff
@@ -31,8 +31,9 @@ import (
 
 var (
 	routineConsoleConfStartCmd = &Command{
-		Path: "/v2/internal/console-conf-start",
-		POST: consoleConfStartRoutine,
+		Path:        "/v2/internal/console-conf-start",
+		POST:        consoleConfStartRoutine,
+		WriteAccess: authenticatedAccess{},
```

Comment:
this probably could be just rootAccess,  cc @anonymouse64

Reply:
mmm yes /v2/internal/console-conf-start is only used by console-conf which runs as root when the user is driving it, so I think this is safe to always be root only and probably makes more sense to only allow it as a root only thing given that it is an internal endpoint